packages/core/src/language-server/glint-language-server.ts

```diff
@@ -88,12 +96,36 @@ export default class GlintLanguageServer {
       directoryExists: this.ts.sys.directoryExists,
       getDirectories: this.ts.sys.getDirectories,
       realpath: this.ts.sys.realpath,
+      useCaseSensitiveFileNames: () => true,
+
+      // @ts-ignore Undocumented method.
+      getCachedExportInfoMap() {
+        // This hook is required so that when resolving a completion item, we can fetch export info
```

Comment:
Thanks for adding nome background here! It's immensely helpful for folks to have inline context like this <3 💪